libkbfs/block_journal.go

```diff
@@ -299,10 +306,20 @@ func (j *blockJournal) appendJournalEntry(
 	return ordinal, nil
 }
 
-func (j *blockJournal) length() (uint64, error) {
+func (j *blockJournal) length() uint64 {
```

Comment:
I think all `blockJournal` methods are on the pointer because it's a biggish type.